__tests__/__fixtures__/requests/multipart-form-data-no-params.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
```

Comment:
The way that this new fixture works with the integration suite is that since it doesn't have `postData.params` in the request entry then the response we get back from https://httpbin.org/anything should match what we have in `response.content`.